lib/model/model.go

```diff
@@ -585,6 +584,7 @@ func (m *Model) FolderStatistics() map[string]stats.FolderStatistics {
 type FolderCompletion struct {
 	CompletionPct float64
 	NeedBytes     int64
+	NeedCount     int64
```

Comment:
What is Count counting? Can we be more specific here (and everywhere it flows)

Reply:
NeedItemCount? That's not much more specific, but I mean it is anything we track - files, dirs and symlinks.

Reply:
`NeedItems` then I think is fine. We have "items" as a word for files-or-folders in a couple of places already, but just a count can be anything.